streams/src/main/java/org/apache/kafka/streams/processor/internals/AssignedTasks.java

```diff
@@ -248,22 +248,22 @@ boolean hasRunningTasks() {
     boolean maybeResumeSuspendedTask(final TaskId taskId, final Set<TopicPartition> partitions) {
         if (suspended.containsKey(taskId)) {
             final Task task = suspended.get(taskId);
-            log.trace("{} found suspended {} {}", logPrefix, taskTypeName, taskId);
+            log.trace("found suspended {} {}", taskTypeName, taskId);
             if (task.partitions().equals(partitions)) {
                 suspended.remove(taskId);
                 task.resume();
                 transitionToRunning(task);
-                log.trace("{} resuming suspended {} {}", logPrefix, taskTypeName, task.id());
+                log.trace("resuming suspended {} {}", taskTypeName, task.id());
                 return true;
             } else {
-                log.trace("{} couldn't resume task {} assigned partitions {}, task partitions", logPrefix, taskId, partitions, task.partitions());
+                log.trace("couldn't resume task {} assigned partitions {}, task partitions", taskId, partitions, task.partitions());
```

Comment:
looks like place holder is missing for the last argument in the earlier code itself.

Reply:
Thanks @omkreddy , fixed this.